src/store/configureStore.js

```diff
@@ -30,6 +36,9 @@ function configureStoreDev(initialState) {
     thunk,
   ];
 
+  const reactRouterMiddleware = routerMiddleware(history);
```

Comment:
I wouldn't push to the array, instead declare the array below all this and just include it ineline.